src/components/progress-circle/progress_circle.html

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 100 100"
+     preserveAspectRatio="xMidYMid meet">
```

Comment:
Could you add a comment that explains why this particular `preserveAspectRatio` value is used?